### PR TITLE
Raise clarity: xpra install on server&client

### PR DESCRIPTION
**Pull request by tux2bsd**

### CODE REVIEW

**Comment by tux2bsd** [comment]
Also, on the main website this most vital part is written as an afterthought.

` This is similar to ssh -X, except that performance will be noticeably better and you can disconnect and re-connect to this session as many times as you like. (obviously xpra must be installed on your server host) `

Remove: `(obviously xpra must be installed on your server host)`

And put `xpra must be installed on your server host` as the FIRST line under "Get started with Xpra".  

This is literally step one so shouldn't be written as an afterthought.

File: README.md [hunk 1]
```diff
@@ -44,16 +44,19 @@ For more details, see [building from source](https://github.com/Xpra-org/xpra/tr
 ---
 
 # Usage
+## Initial requirements
+`xpra` must be installed on the client and the host (version matters).
```
Comment (totaam) [inline README.md:48]: That's not quite true, you can use the html5 client in which case only the server is required...

Comment (tux2bsd) [reply README.md:48]: Try this then:

`xpra` must be installed on the client and the host (version matters).  

You can use the html5 client in which case xpra is only required on the host.  People that are using the html5 client would already be somewhat experienced xpra users.

Comment (tux2bsd) [reply README.md]: I've done another commit.  Just using the github interface which is a bit shit, was only supposed to be a quick adjustment.

File: README.md [hunk 2]
```diff
@@ -44,16 +44,21 @@ For more details, see [building from source](https://github.com/Xpra-org/xpra/tr
...
```
Comment (totaam) [inline README.md:48]: Please remove "(version matters)" or make it a leak to the supported versions wiki page.

File: README.md [hunk 3]
```diff
@@ -44,16 +44,21 @@ For more details, see [building from source](https://github.com/Xpra-org/xpra/tr
 ---
 
 # Usage
+## Initial requirements
+`xpra` must be installed on the client and the host (version matters).  
+
+You can use the html5 client in which case xpra is only required on the host.  People that are using the html5 client would already be somewhat experienced xpra users.
```
Comment (totaam) [inline README.md:50]: Please remove "People..." - this is not helpful or true.

Comment (tux2bsd) [reply README.md]: I've removed it as you don't believe it is helpful.

**Pull request merged**